Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -521,6 +527,8 @@ class BrowserViewController: UIViewController {
         log.debug("BVC calling super.viewDidAppear.")
         super.viewDidAppear(animated)
         log.debug("BVC done.")
+
+        delegate?.browserViewControllerDidAppear(self, profile: profile)
```

Comment:
So right here, just do:

```
let appVersion = AppInfo.appVersion
if profile.prefs.stringForKey("latestAppVersion") != appVersion {
    self.openURLInNewTab(AppConstants.whatsNewURLForVersion(appVersion))
    profile.prefs.setString(appVersion, forKey: "latestAppVersion")
}
```

Note that you only incur the prefs write if the version changed!